_sass/minimal-mistakes/_notices.scss

```diff
@@ -13,17 +13,21 @@
 @mixin notice($notice-color) {
   margin: 2em 0 !important;  /* override*/
   padding: 1em;
-  color: $dark-gray;
+  color: $text-color;
   font-family: $global-font-family;
   font-size: $type-size-6 !important;
   text-indent: initial; /* override*/
-  background-color: mix(#fff, $notice-color, 90%);
+  background-color: mix(#000, $notice-color, 50%);
   border-radius: $border-radius;
   box-shadow: 0 1px 1px rgba($notice-color, 0.25);
 
   h4 {
     margin-top: 0 !important; /* override*/
-    margin-bottom: 0.75em;
+    margin-bottom: 0.75em !important;
+  }
+
+  h4 > code {
```

Comment:
I don't think this is the best solution. I would use a [monospaced element](https://github.com/momentum-mod/docs/blob/2ee6ebb8c69d1af3ca98dcd6343911a79121bf4f/_sass/minimal-mistakes/_base.scss#L151), like kbd, in `single.html` on line 47. Also would have to update the "how to create a docs page" guide with the element as well.

Reply:
Disagreed. `<code>` is semantically correct, `<kbd>` is mostly being used to highlight keyboard or other form of input. I have instead updated the `__base.css` with proper cascading so the code styling only applies when needed, see my latest commit.

Reply:
Better solution 🙂